dashboard/test/helpers/script_levels_helper_test.rb

```diff
@@ -25,20 +25,20 @@ class ScriptLevelsHelperTest < ActionView::TestCase
   test 'script name instead of stage name in header for HOC' do
     self.stubs(:current_user).returns(nil)
     script_level = Script.find_by_name(Script::HOC_NAME).get_script_level_by_chapter 1
-    assert_equal 'Hour of Code', script_level.stage.summarize[:title]
+    assert_equal 'Hour of Code', header_progress(script_level)[:title]
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.